src/main/java/teammates/ui/template/SearchStudentsTable.java

```diff
@@ -8,18 +8,34 @@
  */
 public class SearchStudentsTable {
     private String courseId;
-    private List<StudentRow> studentRows;
+    private List<StudentListSectionData> sections;
+    private boolean hasSection;
     
-    public SearchStudentsTable(String courseId, List<StudentRow> studentRows) {
+    public SearchStudentsTable(String courseId, List<StudentListSectionData> sections) {
         this.courseId = courseId;
-        this.studentRows = studentRows;
+        this.sections = sections;
+        if (sections.size() == 1) {
+            StudentListSectionData section = sections.get(0);
+            if (section.getSectionName().equals("None")) {
+                this.hasSection = false;
+            } else {
+                this.hasSection = true;
+            }
```

Comment:
For the if else statement above, perhaps you could collapse it into just `if (!section.getSectionName().equals("None"))`